src/utils/qrHelpers.js

```diff
@@ -52,6 +52,20 @@ const extractImmunizations = (bundle) => {
   return immunizationResources;
 };
 
+const filterDuplicateImmunizations = (resources) => {
+  // filter immunization resources to those with unique vaccine codes/occurrence dates
+  const filteredResources = resources.filter(
+    (r, index, self) => r.resource.resourceType !== 'Immunization'
+      || self.findIndex(
+        (e) => e.resource.occurrenceDateTime
+        === r.resource.occurrenceDateTime
+        && e.resource.vaccineCode.coding[0].code.coding
+        === r.resource.vaccineCode.coding[0].code.coding
```

Comment:
Should also compare vaccineCode.coding[0].system, since we may get cards with systems other than CVX.